.github/workflows/index-general-search-pr.yml

```
@@ -0,0 +1,111 @@
+      - name: Check out repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Clone docs-internal-data
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          repository: github/docs-internal-data
+          # This works because user `docs-bot` has read access to that private repo.
+          token: ${{ secrets.DOCS_BOT_PAT_READPUBLICKEY }}
+          path: docs-internal-data
+
+      - uses: ./.github/actions/setup-elasticsearch
```

actionlint (1.7.4)

missing input "token" which is required by action "Set up local Elasticsearch" defined at "./.github/actions/setup-elasticsearch". all required inputs are "token"

(action)